extra/sponge.importorder

```diff
@@ -0,0 +1,7 @@
+#Organize Import Order
```

Comment:
The IntelliJ definition is already included in the our formatted settings.
Why do you need to include all these packages? Generally the imports are defined like this:

- Static imports
- All other imports
- `java` imports
- `javax` imports

With each group of the above separated by an empty line.